packages/amplication-client/src/style/mixin.scss

```diff
@@ -468,3 +467,11 @@ $active-indicator-width: 4px;
   line-height: 15px;
   color: #f5f6fa;
 }
+
+@mixin regular-8 {
+  @include font-base();
+  font-weight: 400;
+  font-size: 8px;
+  line-height: 12px;
+  color: #f5f6fa;
```

Comment:
I dont think we should include colors in the mixin for fonts

Reply:
Take a look at this whole file. It has a lot of mixins with a color property
<img width="279" alt="image" src="https://user-images.githubusercontent.com/39680385/187022888-7de339bb-20be-49c8-b11e-c2cccb79100a.png">

Reply:
this is the main color. the rest should run over at the local css file

Reply:
I don't understand, I only added the regular-8 mixin. The rest was there, and they have the color property.

Reply:
I'ts ok